### PR TITLE
Add the forward [ only | first ] option at the global level

### PR DESCRIPTION
**Pull request by jeyglk**
There is already the ability to configure forwarders, but the forward option is missing - therefore adding it.

### CODE REVIEW

**Comment by mmoll** [comment]
@jeyglk could you add a positive and negative spec test, please? :mag:

**Comment by mmoll** [comment]
Oh, and plase push changes into your master branch in this case... there's no need to re-fork and open new a pull request after each change.

Normally, after forking a repo, one does use a topic-branch for such changes and open the PR from that branch. This way it's possible to re-sync with the current master branch from the upstream fork. If you have any questions about the workflow, there's a lot of documentation on help.github.com, but also don't hesitate to ask here!

**Comment by jeyglk** [comment]
@mmoll updated

**Comment by mmoll** [comment]
Maybe adding a validate_re for '(only|first)' would be nice if the variable is set, :+1: otherwise.

**Comment by jeyglk** [comment]
@mmoll Updated again.

File: manifests/init.pp [hunk 1]
```diff
@@ -30,6 +31,9 @@
   validate_re($dns::dnssec_enable, '^(yes|no)$', 'Only \'yes\' and \'no\' are valid values for dnssec_enable field')
   validate_re($dns::dnssec_validation, '^(yes|no|auto)$', 'Only \'yes\', \'no\' and \'auto\' are valid values for dnssec_validation field')
   validate_re($dns::service_ensure, '^running|true|stopped|false$', 'Only \'running\', \'false\', \'stopped\' and \'false\' are validate values for service_ensure field')
+  if $dns::forward {
```
Comment (ekohl) [inline manifests/init.pp]: You can just use $forward here (and the line below).

**Comment by ekohl** [comment]
:+1: with the minor tweak. Would you mind squashing your commits?

File: manifests/init.pp (hunk 1 above)
Comment (jeyglk) [inline manifests/init.pp]: I don't mind modifying it, but the prefix is written everywhere else, I did it to match the other lines.

Comment (ekohl) [inline manifests/init.pp]: You're right, consistency is better.

**Comment by mmoll** [comment]
merged, merci @jeyglk!

**Pull request closed without merging**